### DISCUSSION TITLE
I'm trying to auto generate types on a project and i got an error for the size of the response

### DISCUSSION THREAD

**Discussion by 96erardo** [dotansimha/graphql-code-generator#6386] [Q&A]
I'm trying to generate code for a graphql client and i'm getting a response that the GraphQL schema is above the size allowed (6mb), is there any way to get around this ?

![Captura de pantalla de 2021-07-30 13-48-21](https://user-images.githubusercontent.com/22962147/127692237-96296c28-d1ad-44e1-8b32-54418bee944e.png)

**Comment by dotansimha** [comment]
I'm not sure where this error is coming from. It might be your responsibility from the server? I don't think we have any limitations for file size.
Can you please try to run codegen with `DEBUG=1 VERBOSE=1` and see if you find anything there?

**Comment by dotansimha** [comment]
Are you loading your schema from AWS Lambda? Because it seems like AWS has a limit for 6MB response size - this has nothing to do with codegen (we don't have any limitations on file/schema size).
